node/node.go

```diff
@@ -453,3 +458,76 @@ func (n *Node) checkCurrentNodeIp(ctx context.Context) {
 		}
 	}
 }
+
+// OperatorReachabilityResponse is the response object for the reachability check
+type OperatorReachabilityResponse struct {
+	OperatorID      string `json:"operator_id"`
+	DispersalSocket string `json:"dispersal_socket"`
+	RetrievalSocket string `json:"retrieval_socket"`
+	DispersalOnline bool   `json:"dispersal_online"`
+	RetrievalOnline bool   `json:"retrieval_online"`
+}
+
+func (n *Node) checkNodeReachability() {
+	if n.Config.ReachabilityPollIntervalSec == 0 {
+		n.Logger.Warn("Node reachability checks disabled!!! ReachabilityPollIntervalSec set to 0")
+		return
+	}
+
+	n.Logger.Info("Start nodeReachabilityCheck goroutine in background to check the reachability of the operator node")
+	ticker := time.NewTicker(time.Duration(n.Config.ReachabilityPollIntervalSec) * time.Second)
+	defer ticker.Stop()
+
+	for {
+		<-ticker.C
+
+		if n.Config.DataApiUrl == "" {
+			n.Logger.Error("Unable to perform reachability check - NODE_DATAAPI_URL is not defined in .env")
+			continue
+		}
+
+		checkUrl, err := url.Parse(fmt.Sprintf("%s/api/v1/operators-info/port-check?operator_id=%s", n.Config.DataApiUrl, n.Config.ID.Hex()))
+		if err != nil {
+			n.Logger.Error("Reachability check failed - invalid check url", err, "checkUrl", checkUrl.String())
```

Comment:
Do we want to return in this case or log error and continue checking reachability?

Reply:
My original implementation did that, but I changed the implementation to continuously throw errors to increase the likelihood of operators seeing the error.

Reply:
I think this exits instead of continuously throwing error

Reply:
Good catch.

Reply:
Reverted to original disabled logic